.github/workflows/flutter_integration.yaml

```diff
@@ -61,7 +59,7 @@ jobs:
         uses: reactivecircus/android-emulator-runner@v2
         with:
           api-level: ${{ matrix.api-level }}
+          emulator-options: -no-snapshot-save -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim
           script: |
-            /Users/runner/Library/Android/sdk/tools/bin/avdmanager list
```

Comment:
Is there a reason you removed this 🤔

Reply:
Yes, two reasons to be exact:

1. Because of the switch from `macOS` to `ubuntu` this `avdmanager` path is no longer valid, to he command would fail because of missing `avdmanager`
2. The whole call to `avdmanager list` does nothing useful - it only prints the list of available Android virtual devices to the logs and I failed to find any purpose of it in our CI flow

So I think it's a good idea to simply get rid of it 🙂